Description: Add undeclared source prerequisites
 The package fails to build when using high parallelism (64 threads) due
 to undeclared prerequisites for pdc.
Author: Afif Elghraoui <afif@debian.org>
Bug-Debian: https://bugs.debian.org/846770
Forwarded: https://arc.liv.ac.uk/trac/SGE/ticket/1591
Last-Update: 2016-12-10
--- pkg-gridengine.orig/source/daemons/common/Makefile
+++ pkg-gridengine/source/daemons/common/Makefile
@@ -90,7 +90,7 @@
 test_heartbeat:	test_heartbeat.o qmaster_heartbeat.o $(WINGRIDLIB_DEP) $(UTILIB) $(COMMLISTSLIB)  
 		$(LD_WRAPPER) $(CC) -o $@ $(LFLAGS) test_heartbeat.o qmaster_heartbeat.o $(UTILIB) $(WINGRIDLIB) $(COMMLISTSLIB) $(DLLIB) $(SLIBS) $(LIBS) 
 
-pdc:  $(DAEMONS_COMMON_PDC_OBJS) $(SGEOBJLIB) $(CULLLIB) $(SGEOBJDLIB) $(COMMLIB)
+pdc:  $(DAEMONS_COMMON_PDC_OBJS) $(SCHEDLIB) $(SGEOBJLIB) $(CULLLIB) $(UTILIB) $(UTI2LIB) $(SGEOBJDLIB) $(COMMLIB)
 		$(LD_WRAPPER) $(CC) -o $@ $(LFLAGS) $(DAEMONS_COMMON_PDC_OBJS) $(SCHEDLIB) $(SGEOBJLIB) $(COMMLIB) $(COMMLISTSLIB) $(SGEOBJDLIB) $(CULLLIB) $(UTILIB) $(UTI2LIB) $(WINGRIDLIB) $(DLLIB) $(SECLIB) $(SLIBS) $(LIBS)
 ###############################################################################
 # include dependencies for object files generated by sge_depend
